src/entity/Schema.ts

```diff
@@ -1,32 +1,30 @@
-import {
-    Entity,
-    PrimaryColumn,
-    Column,
-    BeforeInsert,
-    Unique,
-} from 'typeorm';
+import { Entity, PrimaryColumn, Column, BeforeInsert, Unique } from "typeorm";
 import "reflect-metadata";
-import { v4 as uuidv4 } from 'uuid';
-
+import { v4 as uuidv4 } from "uuid";
 
 @Entity()
-
 export class Schema {
-    @BeforeInsert()
-    generateId() {
-        if (!this.id) {
-            this.id = 's' + uuidv4();
-        }
+  @BeforeInsert()
+  generateId() {
+    if (!this.id) {
+      this.id = "s" + uuidv4();
     }
-    @PrimaryColumn()
-    id?: string;
+  }
+  @PrimaryColumn()
+  id?: string;
+
+  @Column({ default: false })
+  registry?: boolean;
+
+  @Column()
+  title?: string;
 
-    @Column({ default: false })
-    registry?: boolean;
+  @Column()
+  description?: string;
 
-    @Column()
-    name?: string;
+  @Column()
+  Ischema?: string;
```

Comment:
Ischema? Why capital `I`?

Reply:
Thought Ischema is better to read than ishema
If you want i can change it
@amarts